### PR TITLE
Works with scalar action

### PR DESCRIPTION
**Pull request by simondlevy**
Awesome repository! I'm using it in my upcoming DRL [course](https://simondlevy.academic.wlu.edu/home/csci-316-advanced-topics-in-robotics/), as well as in my [research](https://github.com/simondlevy/gym-copter).  For this pull request I added a line to the Chapter19 code to support scalar-action environments like Pendulum-v0.

### CODE REVIEW

File: Chapter19/02_play.py [hunk 1]
```diff
@@ -23,7 +24,7 @@
 
     env = gym.make(args.env)
     if args.record:
-        env = gym.wrappers.Monitor(env, args.record)
+        env = wrappers.Monitor(env, args.record)
```
Comment (Shmuma) [inline Chapter19/02_play.py:27]: Not sure about this change. What's the point?

File: Chapter19/02_play.py [hunk 2]
```diff
@@ -38,6 +39,7 @@
         mu_v = net(obs_v)
         action = mu_v.squeeze(dim=0).data.numpy()
         action = np.clip(action, -1, 1)
+        if np.isscalar(action): action = [action]
```
Comment (Shmuma) [inline Chapter19/02_play.py:42]: I'd prefer to stick to pip8. So, this block should look like
```
if np.isscalar(action):
    action = [action]
```

Or, if you prefer to have it one-liner, this one:
```
action = [action] if np.isscalar(action) else action
```

File: Chapter19/01_train_a2c.py [hunk 3]
```diff
@@ -28,30 +28,6 @@
 TEST_ITERS = 100000
 
 
-def test_net(net, env, count=10, device="cpu"):
```
Comment (Shmuma) [inline Chapter19/01_train_a2c.py:31]: Thanks for refactoring those functions! Have missed them somehow :)

File: Chapter19/02_play.py (hunk 1 above)
Comment (simondlevy) [reply Chapter19/02_play.py:27]: https://github.com/dennybritz/reinforcement-learning/issues/63

I get this same issue (under Python3.6.9, Ubuntu 18.04), with successful fix using CurrenJ's solution.  I';m not sure why.

File: Chapter19/02_play.py (hunk 2 above)
Comment (simondlevy) [reply Chapter19/02_play.py]: I agree, and have committed PEP8-compliant version.

**Pull request merged**